server/lease/lessor.go

```diff
@@ -816,13 +816,18 @@ func (le *lessor) initAndRecover() {
 
 // FakeLessor is a fake implementation of Lessor interface.
 // Used for testing only.
-type FakeLessor struct{}
+type FakeLessor struct {
+	LeaseSet map[LeaseID]struct{}
+}
 
 func (fl *FakeLessor) SetRangeDeleter(dr RangeDeleter) {}
 
 func (fl *FakeLessor) SetCheckpointer(cp Checkpointer) {}
 
-func (fl *FakeLessor) Grant(id LeaseID, ttl int64) (*Lease, error) { return nil, nil }
+func (fl *FakeLessor) Grant(id LeaseID, ttl int64) (*Lease, error) {
+	fl.LeaseSet[id] = struct{}{}
+	return nil, nil
+}
 
 func (fl *FakeLessor) Revoke(id LeaseID) error { return nil }
```

Comment:
remove the lease from `LeaseSet`?

Reply:
The existing fake is not really fake, because it doesn't do anything. However, I don't plan implement full Interface if we don't need it.